codebundles/gcp-opssuite-promql/sli.robot

```diff
@@ -0,0 +1,91 @@
+*** Settings ***
+Metadata          Author    Shea Stewart            
+Documentation     Performs a metric query using a PromQL statement on the Ops Suite API
+...               and pushes the result as an SLI metric.
+Force Tags        GCP    OpsSuite    PromQL    Prometheus  Kubernetes
+Library           RW.GCP.OpsSuite
+Library           RW.Core
+Library           RW.Utils
+Library           RW.Prometheus
+Suite Setup       Suite Initialization
+
+*** Tasks ***
+Run Prometheus Instant Query Against Google Prom API Endpoint
+    # Disable logging to hide auth token details
+    Set Log Level    NONE
+    
+    # Get an oauth2 access token from service account json
+    ${token}=    RW.GCP.OpsSuite.Get Token  gcp_credentials=${ops-suite-sa}
```

Comment:
Rather than returning the raw token, couldn't the `Get Token` keyword wrap the return in a `platform.Secret`, effectively doing what line 19 is? This should remove the need for changing the log level.

Reply:
Should be resolved by https://github.com/runwhen-contrib/rw-public-codecollection/pull/32/commits/4313fb30c818f85e637e23618602864281a5233b